StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1203UnitTests.cs

```diff
@@ -386,7 +441,22 @@ protected override IEnumerable<DiagnosticAnalyzer> GetCSharpDiagnosticAnalyzers(
 
         protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
-            return new SA1203SA1214SA1215CodeFixProvider();
+            return new ElementOrderCodeFixProvider();
+        }
+
+        protected override Solution CreateSolution(ProjectId projectId, string language)
+        {
+            Solution solution = base.CreateSolution(projectId, language);
+            if (this.suppressSA1202)
+            {
+                Project project = solution.GetProject(projectId);
+                CompilationOptions options = project.CompilationOptions;
+                ImmutableDictionary<string, ReportDiagnostic> specificOptions = options.SpecificDiagnosticOptions;
+                options = options.WithSpecificDiagnosticOptions(specificOptions.Add(SA1202ElementsMustBeOrderedByAccess.DiagnosticId, ReportDiagnostic.Suppress));
+                solution = solution.WithProjectCompilationOptions(projectId, options);
+            }
+
+            return solution;
         }
```

Comment:
This could be solved by:
```c#
protected override IEnumerable<string> GetDisabledDiagnostics()
{
    if (this.suppressSA1202)
    {
        yield return SA1202ElementsMustBeOrderedByAccess.DiagnosticId;
    }
}